code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -4,6 +4,7 @@
 	color = "#00FF00"
 	var/lootcount = 1		//how many items will be spawned
 	var/lootdoubles = 1		//if the same item can be spawned twice
+	var/nolootchance = 0 // mapper can edit this variable on each of the landmarks so they can decide how rare the loot is.
 	var/list/loot			//a list of possible items to spawn e.g. list(/obj/item, /obj/structure, /obj/effect)
 
 /obj/effect/spawner/lootdrop/New()

```

Comment:
`loot += list("" = nolootchance)`
Put that just inside the if statement in the new proc and take out the blank entries